invenio_assets/assets/build/webpack.config.js

```diff
@@ -24,7 +24,10 @@ var webpackConfig = {
     extensions: ["*", ".js", ".jsx"],
     symlinks: false,
     alias: {
-      "@templates": path.resolve(config.build.context, "templates")
+      "@templates": path.resolve(config.build.context, "templates"),
+      // Semantic-UI LESS config resolving alias
+      // TODO: This would be better injected via config from Flask...
+      "../../theme.config$": path.resolve(config.build.context, "less/invenio_theme/theme.config"),
```

Comment:
I'll extract this into a separate issue since we don't have any quick ideas a the moment on how to avoid it